scipy/stats/tests/test_distributions.py

```diff
@@ -621,6 +621,10 @@ def test_stats(self):
             assert_allclose(s, (2*5.5/1.5) * np.sqrt(2.5/4.5))
             assert_allclose(k, 6*(4.5**3 + 4.5**2 - 6*4.5 - 2)/(4.5*1.5*0.5))
 
+    def test_sf(self):
+        p = stats.pareto.sf(1e9, 2, loc=0, scale=1.5)
+        assert_allclose(p, 2.25e-18)
```

Comment:
"Veridical"--now there's a ten dollar word!  It is just (1.5/1e9)**2.  In general, it is (scale/x)**b.